### PR TITLE
Guess-who-game-Sandra-Wallen

### PR DESCRIPTION
**Pull request by sandra-wallen**

### CODE REVIEW

File: code/script.js [hunk 1]
```diff
@@ -249,95 +251,111 @@ const setSecret = () => {
...
-// This function to start (and restart) the game
+// Function to start the game
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  generateBoard()
+  setSecret()
+
+  console.log(secret)
```
Comment (ElleFrankenberg) [inline code/script.js:260]: I don't think you need to keep the console.log in your code.

File: code/style.css [hunk 2]
```diff
@@ -159,6 +159,10 @@ button {
   height: 100vh;
 }
 
+.show-win-or-lose-section {
+  display: flex;
+}
+
```
Comment (ElleFrankenberg) [inline code/style.css:165]: I don´t know wish way is the best, but I (or rather Isabella how helped me) put this directtielly in the script instead. Like this:

const start = () => { winOrLose.style.display = 'none' board.style.display ='flex'

**Review by ElleFrankenberg — APPROVED** [review]
Hey Sandra😀
As you know my JavaScript knowledge is a bit limited.... With that said I think your code is looking good, the game works and that the blue requirements are fulfilled.
Good job! And thank you again for all the help last week!
Best,
Elle

File: code/script.js (hunk 1 above)
Comment (KarolCislowski) [reply code/script.js:260]: Yes, remember to always clean your code from all console.log()s and commented out code blocks and tips left for you by teachers 🗡️ ✂️

File: code/script.js [hunk 3]
```diff
@@ -249,95 +251,111 @@ const setSecret = () => {
...
+      value: value,
       category: category,
+      alertText: `${value} hair`
     }
   } else if (category === 'eye color') {
-    // Set this up your self
+    currentQuestion = {
+      attribute: 'eyeColor', 
+      value: value,
+      category: category,
```
Comment (KarolCislowski) [inline code/script.js:279]: remember that you do not need to specify the `value` when both sides of the `key: value` pair are the same 🤓

**Review by KarolCislowski — APPROVED** [review]
This is a very nice project 🌈 

I know it was an extra point of this assignment to style it your way but remember that there is no another way to master CSS then practice it over and over again 🦾 

Remember that you can always come back to it when you have some spare time and add more functionalities to make it an even greater portfolio asset! 🦄

**Pull request closed without merging**